src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java

```diff
@@ -205,6 +249,259 @@ public void onResponse(Call call, Response response)
 		});
 	}
 
+	private void sendPlayerRequest(Request request, String target, boolean add)
```

Comment:
I don't think we need a second method for this, just overload sendRequest with a response consumer.